toqito/state_props/entanglement_of_formation.py

```diff
@@ -1,5 +1,6 @@
 """Computes the entanglement of formation of a bipartite quantum state."""
 
+from __future__ import annotations
```

Comment:
BTW I had to re-add this import here. 

Otherwise, the highlighted line below is raised as an error by pylint. 

![image](https://github.com/vprusso/toqito/assets/66048318/fcbb5f0e-5722-4489-bc6c-d3b5e50e815e)

Reply:
Hmm, it might be worthwhile to check what version of Python you are running. Indeed, if we support Python 3.10+ (which I believe is our minimal version), then we shouldn't need to use the `annotations` bit:
https://peps.python.org/pep-0604/

Reply:
Yes, I am using Python 3.11.5. 

It's very interesting pylint does not raise those errors if `from __future__ import annotations` is used. Currently trying figure out why this is so.